### DISCUSSION TITLE
Montage externer LEDs bei Wasseruhr?

### DISCUSSION THREAD

**Discussion by reibuehl** [jomjol/AI-on-the-edge-device#2845] [Q&A]
Ich habe mit der Halterung für die ESP32-Cam von Thingiverse (https://www.thingiverse.com/thing:4573481) das Problem, dass ich wegen Reflexionen von der eingebauten LED kein vernünftiges Bild hin bekomme. Ich würde gerne externe LEDs verwenden, weiß aber nicht, wo ich die am besten platzieren kann. Gibt es da schon Erfahrungen oder vielleicht sogar schon ein 3D Modell?

**Comment by caco3** [comment]
Das kommt extrem auf dein Meter an. Spiel einfach mal damit rum. Das geröt hat einen livestream modus, damit kannst Du in Echtzeit ausprobieren.
